src/06_shared/ui/input/checkbox/roundCheckbox.scss

```diff
@@ -0,0 +1,41 @@
+.round-checkbox-container {
+    position: relative;
+    display: inline-block;
+
+    .round-checkbox {
+        display: none; // Скрываем стандартный чекбокс
+    }
+
+    .checkbox-label {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 17px; // Ширина чекбокса
+        height: 18px; // Высота чекбокса
+        border-radius: 50%; // Делаем круглый
+        background-color: var(--clr-pale-grey); // Начальный цвет фона
+        cursor: pointer;
+        border: 1px solid rgb(var(--checkbox-border));
+        transition: background-color 0.3s ease;
+
+        .checkbox-icon {
+            width: 9px; // Ширина изображения
+            height: 10px; // Высота изображения
+            transition: filter 0.3s ease; // Плавный переход для фильтра
+            
+            path {
+                stroke: rgb(var(--checkbox-icon));/* Цвет по умолчанию */
+            }
```

Comment:
when you use svg sprite, you can manage svg colour by color: style

Reply:
it doesn't work when i don't use stroke